etcdserver/server.go

```diff
@@ -370,6 +370,7 @@ func (s *EtcdServer) run() {
 				}
 				s.r.raftStorage.ApplySnapshot(rd.Snapshot)
 				snapi = rd.Snapshot.Metadata.Index
+				confState = rd.Snapshot.Metadata.ConfState
```

Comment:
I think confState should be updated at line 389.
confState is updated together with appliedi, and is used to apply conf change entry.